src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -848,6 +851,9 @@ private void FlushPendingOutput()
         /// Name of the user whose credential is to be prompted for. If set to null or empty
         /// string, the function will prompt for user name first.
         /// </param>
+        /// <param name="reEnterPassword">
+        /// Promts user to re-enter the password for confirmation
```

Comment:
Please add final dot in comment:
```suggestion
        /// Prompts user to re-enter the password for confirmation.
```